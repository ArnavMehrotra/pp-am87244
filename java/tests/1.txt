A B
56 2
57 100
1 4