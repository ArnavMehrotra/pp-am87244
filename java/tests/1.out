__to_upper__([1,0]) 
qqqqq 
